os/binfmt/binfmt_unloadmodule.c

```diff
@@ -143,7 +144,7 @@ int unload_module(FAR struct binary_s *binp)
 		if (binp->unload) {
 			ret = binp->unload(binp);
 			if (ret < 0) {
-				berr("binp->unload() failed: %d\n", ret);
+				berr("%s binp->unload(): %d\n", clog_message_str[CMN_LOG_FAILED_OP], ret);
```

Comment:
which is good 
1. operation failure binp->unload()  or 
2. binp->unload() operation failure ?

Reply:
we have followed operation failure binp->unload() pattern at all places.